widget/src/text_input.rs

```diff
@@ -1853,3 +1856,32 @@ fn alignment_offset(
         }
     }
 }
+
+#[cfg(target_os = "macos")]
+fn convert_macos_shortcut(
+    key: &keyboard::Key,
+    modifiers: keyboard::Modifiers,
+) -> &keyboard::Key {
+    let control = modifiers.control();
+    match key.as_ref() {
+        keyboard::Key::Character("b") if control => {
```

Comment:
this also catches ctrl + alt or any other combination of modifiers that uses ctrl, which is probably not intended

Reply:
Thanks. I fixed this and force-pushed the fix to this branch (with rebasing it onto the latest master branch).